src/components/OptionsSelector.js

```diff
@@ -87,7 +92,13 @@ class OptionsSelector extends Component {
     }
 
     componentDidMount() {
-        this.textInput.focus();
+        this.unsubscribeTransitionEnd = this.props.navigation.addListener('transitionEnd', () => {
```

Comment:
What transition are we listening for the end of here? 
Does this work for all screens that might use this component in the future?
Can we add some context about why this change was needed so that others have a better idea whether to follow this example or not?

Reply:
Sure. We used this as Interaction Manager has issues on Web. This transition waits for the Navigation transition to finish. This component is being used for only Screen components so this works fine. But it's a good idea to mention it in a comment. Oh, PR is merged. I will open follow-up PR.

Let me know If we should handle this in a different way.